app/dashboard/organization/overview/page.tsx

```diff
@@ -1,11 +1,213 @@
+            const {userUUID, orgID} = await getUserIDandOrgID();
+            setUserID(userUUID)
+            setUserOrgID(orgID);
+
+            if(orgID === null){
+                router.push("/dashboard/organization/overview/orgSignup")
+            }
+
+            getUserOrgData(orgID).then(({data}) => setOrgData(data[0])).catch((error) => (displayErrorToast(error)));
+            isUserOwner(userUUID, orgID).then(setIsOwner);
+            getOrgTeam(orgID).then(setOrgTeam);
+        }
+
+        getAllData();
+    }, [])
+
+    useEffect(() => {
+        setOrgName(orgData?.name || "");
+        setOrgEmail(orgData?.email || "");
+        setOrgBio(orgData?.bio || "");
+        setOrgWebsite(orgData?.website || "");
+    }, [orgData])
+
+    // For loading screen
+    useEffect(() => {
+        if(orgData !== null &&  userOrgID !== -1 && orgTeam !== null){
+            setLoading(false);
+        }
+    }, [orgData, userOrgID, orgTeam])
+
+    const handleUpdateOrg = async () => {
+        if(orgName == "" || orgEmail == ""){
+            setFormError(true);
+            return;
+        }
+        setLoading(true)
+        const error = await updateOrg(orgName, orgEmail, orgBio, orgWebsite, userOrgID);
+
+        if(error){
+            displayErrorToast(error);
+        }
+        else {
+            displaySuccessToast()
+        }
+        setLoading(false);
+    }
+
+    const displayErrorToast = (error) => {
+        console.log(error.message);
+        toast({
+            variant: "destructive",
+            title: "Unable to update organization",
+            description: "Check log for error",
+        })
+    }
+
+    const displaySuccessToast = () => {
+        toast({
+            title: "Organization Successfully Updated",
+            })
+    }
+
+    const leaveOrg = async () => {
+        const error = await removeUserFromOrg(userID);
+
+        if (error){
+            console.log(error.message);
+            toast({
+            variant: "destructive",
+            title: "Unable to leave org",
+            description: "Check log for error",
+            })
+        }
+        else{
+            toast({
+            title: "Successfully Left Organization",
+            })
+            router.push("/dashboard/organization/overview/orgSignUp");
+        }
+    }
     return(
         <div className="">
-            <PageHeading header="Overview" description="Your are currently not part of any organization"/>
+            <PageHeading header="Overview" description="Your organization"/>
+            {loading ? 
+            <div className="text-center 2xl:mt-72 mt-52 text-darkmaroon">
+                <div className="text-xl font-medium">Getting organization data</div>
+                <div className="loadingAnimation"><div></div><div></div><div></div></div>
+            </div> 
+            :
+            <div>
+            <div className="flex items-stretch mt-8 2xl:space-x-6 > * + * space-x-4 > * + * animate-in">
```

Comment:
What does > * + * do? I can't find any documentation about it in Tailwind.

Reply:
Space between